components/atom/input/src/InputWrapper.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import InputLabel from './Label'
+
+const InputWrapper = (props) => {
+  const {name, label, children} = props
+  return (
+    <div className='sui-AtomInput'>
```

Comment:
I think the best practice is to enclose inputs inside their label for accessibility. Maybe Pablo has a better opinion.

Reply:
It's ok as long as you use the label `for` attribute. Your approach is recommended if it's not possible to set the `for` attribute

More info here https://www.w3.org/WAI/tutorials/forms/labels/ where you can read that `Generally, explicit labels are better supported by assistive technology.`